### DISCUSSION TITLE
Fluent loading error while opening through Pyfluent

### DISCUSSION THREAD

**Discussion by nakul-07** [ansys/pyfluent#1836] [Q&A]
Fluent is quitting as soon as in opens if loaded through PyAnsys, Is there a way to correct this?

**Comment by mkundu1** [comment]
Please share the versions of PyFluent and Fluent and how Fluent is launched from PyFluent.

**Reply by nakul-07** [reply]
I have tried all the versions of PyFluent from 0.16.0 to 0.17.dev2, and ansys version 22R2

following message is coming after starting a fluent session
....
...
Cleanup script file is C:\\Working_folder\\ANSYS\\Pyfluent\\cleanup-fluent-N0228-4896.bat

Posting ANSYS Product Improvement Program startup data
Done.

> 
Posting ANSYS Product Improvement Program performance data 
Done.
Shutting down Server...
Services Shutdown...Done.
Server Shutdown...Done.
Queue Cleared...Done.
Shutdown Complete.
Auto-Transcript Stop Time:  01:37:44, 07 Aug 2023 GMT Summer Time 
Total Transcript Time: 0 Minute 27 Second.

**Reply by raph-luc** [reply]
@nakul-07 I can't reproduce that issue. Your Fluent transcript output indicates that the session is being closed, and that is probably being caused by your Python interpreter exiting.

If you are launching Fluent using PyFluent, and do not want to use PyFluent/Python itself, set `cleanup_on_exit=False` for your `launch_fluent` command

See the documentation: https://fluent.docs.pyansys.com/version/stable/api/general/launcher/index.html#ansys.fluent.core.launcher.launcher.launch_fluent
